README.md

```diff
@@ -3,4 +3,8 @@ CoreLogic Weather Verification Services Wrapper
 
 [![Build status](https://ci.appveyor.com/api/projects/status/0xdnvm6x9lx37yp2?svg=true)](https://ci.appveyor.com/project/Georadix/corelogic-wvs)
 
-Copyright &copy; 2016 [CoreLogic](http://corelogic.com).
\ No newline at end of file
+Code Coverage Reports:
+- [Develop](https://ci.appveyor.com/api/projects/Georadix/corelogic-wvs/artifacts/CodeCoverage.zip?branch=develop)
+- [Latest Build](https://ci.appveyor.com/api/projects/Georadix/corelogic-wvs/artifacts/CodeCoverage.zip)
```

Comment:
Do we want to keep the latest build or just go with develop? To check a PR, easiest thing will be to click on build details which will bring us to AppVeyor and have artifacts there.